Source/Scene/GroundPrimitive.js

```diff
@@ -205,6 +223,7 @@ define([
         this._readyPromise = when.defer();
 
         this._primitive = undefined;
+        this._debugPrimitive = undefined;
```

Comment:
Does this get destroyed when `destroy` is called or when `debugShowShadowVolume` is false?